activation/post.go

```diff
@@ -27,6 +27,8 @@ type PostConfig struct {
 	LabelsPerUnit uint64 `mapstructure:"post-labels-per-unit"`
 	K1            uint32 `mapstructure:"post-k1"`
 	K2            uint32 `mapstructure:"post-k2"`
+	B             uint32 `mapstructure:"post-b"`
```

Comment:
Add test to check that `B` is always 16, since we are currently not allowing other values (although the implementation could handle it).